okhttp/src/main/java/okhttp3/internal/http/HttpHeaders.java

```diff
@@ -144,53 +181,116 @@ public static Headers varyHeaders(Headers requestHeaders, Headers responseHeader
   }
 
   /**
-   * Parse RFC 7617 challenges, also wrong ordered ones.
-   * This API is only interested in the scheme name and realm.
+   * Parse RFC 7235 challenges.
+   * If a challenge uses the {@code token68} variant instead of auth params,
```

Comment:
this doc is duplicated from `Response.challenges()`. Since this class isn’t public API, maybe just omit it?